app/views/registrations_mailer/notify_organizers_of_deleted_registration.html.erb

```diff
@@ -3,7 +3,7 @@
   <% if @registration.user.wca_id %>
     (<%= wca_id_link @registration.user.wca_id -%>)
   <% end %>
-  just deleted their registration for <%= @registration.competition.name %>.
+  just deleted their registration for <%= link_to @registration.competition.name, competition_url(@registration.competition.id) %>.
```

Comment:
This is the mail that notifies organizers, so they have access to the registrations panel. Do we really want to link to the competition home page? Linking to the page where they can admin registrations makes more sense in my mind